packages/plugin-core/src/components/views/LookupV3QuickPickView.ts

```diff
@@ -0,0 +1,303 @@
+            if (ToLinkBtn) ToLinkBtn.pressed = false;
+            if (ExtractBtn) ExtractBtn.pressed = false;
+            if (ToItemsBtn) ToItemsBtn.pressed = true;
+            break;
+          }
+
+          case SelectionMode.selection2Link: {
+            if (ToLinkBtn) ToLinkBtn.pressed = true;
+            if (ExtractBtn) ExtractBtn.pressed = false;
+            if (ToItemsBtn) ToItemsBtn.pressed = false;
+            break;
+          }
+          case SelectionMode.selectionExtract: {
+            if (ToLinkBtn) ToLinkBtn.pressed = false;
+            if (ExtractBtn) ExtractBtn.pressed = true;
+            if (ToItemsBtn) ToItemsBtn.pressed = false;
+            break;
+          }
+          case SelectionMode.None: {
+            if (ToLinkBtn) ToLinkBtn.pressed = false;
+            if (ExtractBtn) ExtractBtn.pressed = false;
+            if (ToItemsBtn) ToItemsBtn.pressed = false;
+            break;
+          }
+          default:
+            assertUnreachable(newValue);
+        }
+
+        const buttons: DendronBtn[] = [];
+
+        if (ToLinkBtn) buttons.push(ToLinkBtn);
+        if (ExtractBtn) buttons.push(ExtractBtn);
+        if (ToItemsBtn) buttons.push(ToItemsBtn);
+
+        this.updateButtonsOnQuickPick(...buttons);
+      })
+    );
+
+    // Vault Selection is mapped to 'other' for some reason; Fix Later.
+    const vaultSelectionBtn = this.getButton("other");
+    if (vaultSelectionBtn !== undefined) {
+      this._disposables.push(
+        this._viewState.vaultSelectionMode.bind(async (newValue) => {
+          vaultSelectionBtn.pressed =
+            newValue === VaultSelectionMode.alwaysPrompt;
+          this.updateButtonsOnQuickPick(vaultSelectionBtn);
+        })
+      );
+    }
+
+    const multiSelectBtn = this.getButton("multiSelect");
+    if (multiSelectBtn) {
+      this._disposables.push(
+        this._viewState.isMultiSelectEnabled.bind(async (newValue) => {
+          multiSelectBtn.pressed = newValue;
+          this.updateButtonsOnQuickPick(multiSelectBtn);
+        })
+      );
+    }
+
+    const copyLinkBtn = this.getButton("copyNoteLink");
+    if (copyLinkBtn) {
+      this._disposables.push(
+        this._viewState.isCopyNoteLinkEnabled.bind(async (enabled) => {
+          copyLinkBtn.pressed = enabled;
+          this.updateButtonsOnQuickPick(copyLinkBtn);
+        })
+      );
+    }
+
+    const directChildBtn = this.getButton("directChildOnly");
+    if (directChildBtn) {
+      this._disposables.push(
+        this._viewState.isApplyDirectChildFilter.bind(async (newValue) => {
+          directChildBtn.pressed = newValue;
+          this.updateButtonsOnQuickPick(directChildBtn);
+        })
+      );
+    }
+
+    const journalBtn = this.getButton(LookupNoteTypeEnum.journal);
+    const scratchBtn = this.getButton(LookupNoteTypeEnum.scratch);
+    const taskBtn = this.getButton(LookupNoteTypeEnum.task);
+
+    if (journalBtn && scratchBtn && taskBtn) {
```

Comment:
`CreateJournalNoteCommand` and `CreateScratchNoteCommand` both creates their own custom controller and uses those. I think with this block, the controller for `CreateJournalNoteCommand` and `CreateScratchNoteCommand` will not function correctly.

Reply:
To be fair, I think the implementation of both is a bit hacky at best. Would they need their own separate `QuickPickView`s?

Reply:
Good call - yeah I just amended this object to do a bunch of `if(journalBtn) then set/unset it`. 

I agree it's hacky - I think ideally we add args to this view's constructor to instantiate the view with different options based on which buttons are active.  But right now, that button status is only exposed via a prop in `DendronQuickPickerV2`, so it's a bit messy.  I think I'll keep it for now - there are changes I do wanna make later to clean up how the Controller/Provider interfaces work, but to reduce scope for this particular change and focus on circ dep removal, I'll defer that for now.